vl-test.js

```diff
@@ -0,0 +1,42 @@
+import { css, html, LitElement } from 'https://unpkg.com/lit-element@latest/lit-element.js?module';
+import {VlElement, define} from '/node_modules/vl-ui-core/vl-core.js';
+import './vl-modal.src.js';
+import '/node_modules/vl-ui-button/vl-button.js';
+import '/node_modules/vl-ui-input-field/vl-input-field.js';
+
+export class VlTest extends LitElement {
+
+
+  constructor() {
+    super();
+  }
+
+  render() {
+    return html`
+            <style>
+                @import '../style.css';
+                @import '/node_modules/vl-ui-icon/style.css';
+                @import '/node_modules/vl-ui-link/style.css';
+                @import '/node_modules/vl-ui-action-group/style.css';
+                @import '/node_modules/vl-ui-button/style.css';
+                @import '/node_modules/vl-ui-input-field/style.css';
+            </style>
+    <script src="/vl-modal.src.js"></script>
+
+    <script src="/node_modules/vl-ui-button/vl-button.js"></script>
+
+    <script src="/node_modules/vl-ui-action-group/vl-action-group.js"></script>
+
+    <script src="/node_modules/vl-ui-icon/vl-icon.js"></script>
+
+                <button is="vl-button" data-vl-modal-open="modal-3" onclick="openModalTest(this, 'modal-3')">Open</button>
```

Comment:
modal-3 wordt al gebruikt in de demo pagina en bij een klik op de button worden de 2 modals geopend. Als dit moet behouden blijven, best een andere (specifiekere) naam gebruiken.